src/guides/v2.3/migration/extend-the-tool.md

```diff
@@ -62,20 +62,18 @@ The following shows an example of using both mapping rules and a handler. This e
 </destination>
 ```
 
-Refer to the following for an explanation of the changes in the previous example:
-
--  Unnecessary data from the index table `great_blog_index` should not be migrated
--  The table `great_blog_publication` was renamed to `great_blog_post` in Magento 2, so data should be migrated to the new table
-   -  The `summary` field was renamed to `title`, so data should be migrated to the new field
-   -  The `priority` field was removed and no longer exists in Magento 2
-   -  The data in the `body` field has changed format and should be processed by the custom handler: `\Migration\Handler\GreatBlog\NewFormat`
--  A new ratings feature was developed for the "GreatBlog" extension in Magento 2
-   -  A new `great_blog_rating` table was created
-   -  A new `great_blog_post.rating` field was created
+-  Unnecessary data from the `great_blog_index` index table should not be migrated.
+-  The table `great_blog_publication` was renamed to `great_blog_post` in Magento 2, so data will be migrated to the new table.
+   -  The `summary` field was renamed to `title`, so data will be migrated to the new field.
+   -  The `priority` field was removed and no longer exists in Magento 2.
+   -  The data in the `body` field has changed format and should be processed by the custom handler: `\Migration\Handler\GreatBlog\NewFormat`.
+-  A new ratings feature was developed for the "GreatBlog" extension in Magento 2.
+   -  A new `great_blog_rating` table was created.
+   -  A new `great_blog_post.rating` field was created.
 
 ### Extend mapping in other steps
 
-Other steps support mapping, such as the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav) and Customer Attributes Step. These steps migrate a predefined list of Magento tables. For example, suppose that the "GreatBlog" extension has an additional field in the `eav_attribute` table and the name changed in Magento 2. Since the table is processed by the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav), mapping rules should be written for the `map-eav.xml` file. The `map.xml` and `map-eav.xml` files use the same `map.xsd` schema, so mapping rules remain the same.
+Other steps support mapping, such as the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav) and the Customer Attributes Step. These steps migrate a predefined list of Magento tables. For example, suppose that the "GreatBlog" extension has an additional field in the `eav_attribute` table and the name changed in Magento 2. Since the table is processed by the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav), mapping rules should be written for the `map-eav.xml` file. The `map.xml` and `map-eav.xml` files use the same `map.xsd` schema, so mapping rules remains the same.
```

Comment:
```suggestion
Other steps support mapping, such as the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav) and the Customer Attributes Step. These steps migrate a predefined list of Magento tables. For example, suppose that the "GreatBlog" extension has an additional field in the `eav_attribute` table and the name changed in Magento 2. Since the table is processed by the [EAV Step]({{ page.baseurl }}/migration/migration-tool-internal-spec.html#eav), mapping rules should be written for the `map-eav.xml` file. The `map.xml` and `map-eav.xml` files use the same `map.xsd` schema, so mapping rules remain the same.
```